Explorer workshop guide

MichelleBaird-DBT opened the pull request:
## What are you changing in this pull request and why?
Adding an explorer guide. To be used in the coalesce explorer workshop and then later in an online explorer workshop a la zero to dbt.

[Preview link](https://docs-getdbt-com-git-explorer-workshop-guide-dbt-labs.vercel.app/guides/explorer-quickstart?step=1)

## Checklist
- [ x] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ nope] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ x] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Note: Will probably submit an additional pr on oct 1st once changes to dbt explorer are in preview. Will be replacing images with updated images. 

Additional note: Kyle and Shelli, please give this pr a technical and pedagogical review. Mirna, please let me know if I've formatted everything here correctly. I used the sl qs as a guide. 

ADDING OR REMOVING PAGES (if so, uncomment):
- [? ] Add/remove page in `website/sidebars.js`
- [ x] Provide a unique filename for new pages
- [ ?] Add an entry for deleted pages in `website/vercel.json`
- [ ?] Run link testing locally with `npm run build` to update the links that point to deleted pages
Added question marks for steps I'm unsure of. Mirna, could you let me know if there's anything else I should be doing before we push to prod?

MichelleBaird-DBT committed:
Create explorer-qs.md

added intro

MichelleBaird-DBT committed:
Update explorer-qs.md

added setup and part of first portion

MichelleBaird-DBT committed:
Update explorer-qs.md

added everything but images

MichelleBaird-DBT committed:
Add files via upload

MichelleBaird-DBT committed:
Add files via upload

MichelleBaird-DBT committed:
Add files via upload

MichelleBaird-DBT committed:
Add files via upload

MichelleBaird-DBT committed:
Update explorer-qs.md

added images

MichelleBaird-DBT committed:
Update explorer-qs.md

removed sl guide imports

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,98 @@
+dbt Explorer provides a visualization of your project’s [DAG](/terms/dag) that you can interact with. The nodes in the lineage graph represent the project’s resources and the edges represent the relationships between the nodes. Nodes are color-coded and include iconography according to their resource type.
+
+- Use the search bar and [node selectors](/reference/node-selection/syntax) to filter your dag.
+- [Lenses](docs/collaborate/explore-projects#lenses) make it easier to understand your project’s contextual metadata at scales, especially to distinguish a particular model or a subset of models.
```
```suggestion
- [Lenses](/docs/collaborate/explore-projects#lenses) make it easier to understand your project’s contextual metadata at scales, especially to distinguish a particular model or a subset of models.
```

mirnawong1 committed:
Update website/docs/guides/explorer-qs.md

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,98 @@
+---
+title: "Quickstart for the dbt Explorer Workshop"
```
```suggestion
title: "Quickstart for the dbt Explorer workshop"
```

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,98 @@
+---
+title: "Quickstart for the dbt Explorer Workshop"
+id: ex-qs
```
```suggestion
```

mirnawong1 committed:
Update website/docs/guides/explorer-qs.md

mirnawong1 committed:
Update website/docs/guides/explorer-qs.md

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,97 @@
+  - If you are already signed into dbt Cloud, sign out first!
+  - Enter your first name and last name
+  - Choose the “Exploring a dbt Mesh implementation with dbt Explorer” option.
+- Agree to the terms of service and click the “Complete Registration” button. Wait about 30 seconds, and you’ll be in the dbt Cloud project for this course, already connected to a data warehouse!
```
there's also a password option - i think we should mention they can enter the password given to them at the workshop (i assume)?

mirnawong1 replied:
@MichelleBaird-DBT this is what i saw:

![Screenshot 2024-09-26 at 11 22 22](https://github.com/user-attachments/assets/374a35fe-38d4-4bbc-8d53-649e328ca817)

mirnawong1 committed:
update first page

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,105 @@
+
+#### Hands-On
+- Explore the Model tab 
+  - Which job took the Longest? 
```
how do you feel about 'Try it out' so it's more clear that this is an interactive part?

mirnawong1 committed:
tweaks

mirnawong1 committed:
add id

mirnawong1 committed:
add next steps

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,131 @@
+### Hands-On
+- Review your project recommendations and choose one improvement to implement (such as adding a description to a primary key that’s missing one).
+
+## What's next
```
@MichelleBaird-DBT added a 'what's next' - what do you think? also, would this be a good place to write about the 'coming soon or new features (such as aut exposures, model query history, and data health tiles)?

MichelleBaird-DBT replied:
I think so! Adding this now

mirnawong1 reviewed (approved):
hey @MichelleBaird-DBT , this looks great! thank you for opening this up and made some small tweaks for you, but other than that it looks good to me! i have some questions that i posed but non are blockers. incredible job!

MichelleBaird-DBT reviewed (commented):
Changes look good, thanks for the review!

MichelleBaird-DBT committed:
Update explorer-qs.md

Added section for future improvements to explorer. Did not add bullet points for features that'd be in preview by Coalesce time. Used this slack message from Roxi as a ref https://dbt-labs.slack.com/archives/CSSHB76QL/p1727296314616569?thread_ts=1727291542.371399&cid=CSSHB76QL

MichelleBaird-DBT committed:
Update explorer-qs.md

capitalization consistent

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+
+## Introduction
+
+Unlock the power of [dbt Explorer](/docs/collaborate/explore-projects) in this hands-on workshop designed for analytics engineers, data analysts, stakeholders, and data leaders.
```
suggest a contnet change here
```suggestion
Test test Unlock the power of [dbt Explorer](/docs/collaborate/explore-projects) in this hands-on workshop designed for analytics engineers, data analysts, stakeholders, and data leaders.
```

swhite-dbt commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+  - Sign out if you're already logged in.
+  - Enter your first name and last name.
+  - Select the **Exploring a dbt Mesh implementation with dbt Explorer** option.
+  - Use the password provided by the workshop facilitator.
```
Can we use passcode since that is the verbiage for the signup page?

mirnawong1 replied:
```suggestion
  - Use the passcode provided by the workshop facilitator.
```

swhite-dbt commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+  - What sources can we see?
+
+- Explore the **Tests** Tab
+  - Did we run the right test?, Which test were missing? Why?
```
Extra comma here

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

mirnawong1 committed:
Update website/docs/guides/explorer-qs.md

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+  - What sources can we see?
+
+- Explore the **Tests** Tab
+  - Did we run the right test?, Which test were missing? Why?
```
```suggestion
  - Did we run the right test? Which tests were missing? Why?
```

mirnawong1 committed:
Update website/docs/guides/explorer-qs.md

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+
+Keep an eye out for new features coming out soon, like:
+- More auto exposure integrations (like PowerBI)
+- Model query history for additional warehouses (like Redshift and DataBricks)
```
```suggestion
- Model query history for additional warehouses (like Redshift and Databricks)
```

mirnawong1 committed:
Update website/docs/guides/explorer-qs.md

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+For the next steps, you can check out the [dbt Explorer documentation](/docs/collaborate/explore-projects) and [FAQs](/docs/collaborate/dbt-explorer-faqs) to learn more about how to use dbt Explorer.
+
+Keep an eye out for new features coming out soon, like:
+- More auto exposure integrations (like PowerBI)
```
```suggestion
- More [auto-exposure](/docs/collaborate/auto-exposures) integrations (like PowerBI).
```

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+
+Keep an eye out for new features coming out soon, like:
+- More auto exposure integrations (like PowerBI)
+- Model query history for additional warehouses (like Redshift and Databricks)
```
```suggestion
- [Model query history](/docs/collaborate/model-query-history) for additional warehouses (like Redshift and Databricks)
```

mirnawong1 commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+Keep an eye out for new features coming out soon, like:
+- More auto exposure integrations (like PowerBI)
+- Model query history for additional warehouses (like Redshift and Databricks)
+- Improvements to data health tiles
```
```suggestion
- Improvements to [data health tiles](/docs/collaborate/data-tile)
```

mirnawong1 committed:
Update website/docs/guides/explorer-qs.md

mirnawong1 committed:
Update website/docs/guides/explorer-qs.md

mirnawong1 committed:
Update website/docs/guides/explorer-qs.md

mirnawong1 committed:
Update explorer-qs.md

mirnawong1 commented:
thanks @swhite-dbt ! have folded in your feedback! @MichelleBaird-DBT have taken another pass and it's good on my end. is there any one else you'd like to have review?

coapacetic commented:
taking a quick look now team 👀

coapacetic commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+
+## Setup
+Now we’ll be creating your dbt Cloud account and connecting it to a data warehouse. 
+1. Go to this URL:  https://cloud.getdbt.com/coalesce-workshop-signup:
+  - Sign out if you're already logged in.
```
```suggestion
1. Go to this URL:  https://cloud.getdbt.com/coalesce-workshop-signup (Sign out if you're already logged in)
```
formatting here looked funky and extra colon at the end of the URL

coapacetic commented on website/docs/guides/explorer-qs.md:
```diff
@@ -0,0 +1,137 @@
+<Lightbox src="/img/quickstarts/dbt-cloud/dbt_explorer_project_recs.png" width="80%" title="dbt Explorer's project recommendation tab" />
+
+### Hands-On
+- Review your project recommendations and choose one improvement to implement (such as adding a description to a primary key that’s missing one).
```
Do you want to leave this open ended to a the entire audience or guide them to a specific change?

If they are going to make a specific change, you should add the steps for making a change (go to the IDE, open a branch, etc. etc.).

Otherwise you leave yourself open to everyone trying something different and potential chaos

coapacetic reviewed (changes requested):
Two comments for your review @MichelleBaird-DBT 

One big one, so requesting changes

MichelleBaird-DBT committed:
Update explorer-qs.md

fixed weird bullet indent

MichelleBaird-DBT committed:
Update explorer-qs.md

updated hands on bullets

MichelleBaird-DBT committed:
Add files via upload

MichelleBaird-DBT committed:
Add files via upload

MichelleBaird-DBT committed:
Update explorer-qs.md

add images for clarity when running jobs

MichelleBaird-DBT committed:
Add files via upload

MichelleBaird-DBT committed:
Update explorer-qs.md

added clarifying image

MichelleBaird-DBT committed:
Merge branch 'current' into explorer-workshop-guide

MichelleBaird-DBT committed:
Update explorer-qs.md

fixed performance hands on

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

mirnawong1 committed:
Update explorer-qs.md

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

mirnawong1 commented:
hey @MichelleBaird-DBT @coapacetic @swhite-dbt - just checking if you need any help with this PR?

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

coapacetic committed:
coap tweaks in final review

coapacetic committed:
Merge branch 'current' into explorer-workshop-guide

coapacetic committed:
tweak image and some copy

coapacetic committed:
Merge branch 'explorer-workshop-guide' of github.com:dbt-labs/docs.getdbt.com into explorer-workshop-guide

coapacetic reviewed (approved):
made some very quick tweaks live for you and looks good to me on my local machine

send it after all the checks pass

MichelleBaird-DBT committed:
Merge branch 'current' into explorer-workshop-guide

MichelleBaird-DBT commented:
@mirnawong1 Mind taking a look? Made some minor updates

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide

mirnawong1 commented:
merging now team!

mirnawong1 committed:
Update explorer-qs.md

mirnawong1 committed:
Merge branch 'current' into explorer-workshop-guide
